README.md

```diff
@@ -1 +1,47 @@
-# Capstone-Project
\ No newline at end of file
+# Capstone Project
+
+> Microverse first capstone project for first module. Project has mobile and desktop version project which have files of html, css and javascript as well as correcting using linters. Code is deployed on Github pages.
+
+# Website Link
+- 
+
+## Built With
+
+- html
+- css
+- javascript
+
+## Authors
+
+- Name: Muhammad Umair Muqeem
+- GitHub: [@umair-rao](https://github.com/umair-rao)
+- Twitter: [@umair_ra0](https://twitter.com/umair_ra0)
+- LinkedIn:(https://www.linkedin.com/in/engr-umair-muqeem/)
+
+
+## List of technologies
+
+ - Visual Studio Code
+ - Github
+ 
+## Instuctions to set up project locally
+ - Download zip file of the project
+ - Extract folder to the hard drive
+ - Opened index.html file with any modern web browser i.e. chrome
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+Cindy Shin the author of the original design [https://www.behance.net/gallery/29845175/CC-Global-Summit-2015]
+
+##Licence
+ - This project is [MIT](./license) licensed.
```

Comment:
[OPTIONAL] The link to the license file is not working because it is in small letters while the file name is in capital case. Kindly check and correct it. :pray:

Reply:
corrected.